app/api-v1/routes/recipe.js

```diff
@@ -5,7 +5,27 @@ module.exports = function (recipeService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      if (!req.body) {
+        return res.status(400).json({ message: 'Invalid request' })
+      }
+
+      const { id, externalId, name, imageAttachmentId, material, alloy, price, requiredCerts, supplier } = req.body
```

Comment:
need to ensure `imageAttachmentId` is valid as well as `supplier` is a valid name in the `dscp-identity-service`

Reply:
are those separate to this? nothing like that functionality exists, small prs etc...

Reply:
don't miind this going in without that but this ticket isn't done yet. That was really the point